managed-vpc/build.f/particles-initial/config/default.js

```diff
@@ -3,7 +3,7 @@ var config = {
     {
       aws: {
         region: 'us-east-1',
-        bucket: 'sgas.particles-managed-vpc.714270045944.us-east-1'
+        bucket: 'sgas.particles-managedvpc.714270045944.us-east-1'
```

Comment:
Because sgas.particles-managed-vpc.714270045944.us-east-1 bucket name is not available.